MODULE.bazel

```diff
@@ -190,7 +190,7 @@ erlang_package.hex_package(
         "rm ebin/cowboy.app",
     ],
     sha256 = "0fa395437f1b0e104e0e00999f39d2ac5f4082ac5049b67a5b6d56ecc31b1403",
```

Comment:
If the hash is unchanged, this will produce an incorrect cache it. Comment out the line, build with `bazel build @cowboy//:erlang_app` and the new sha should be printed. Then you can update it.